hpoflow/optuna_transformers.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2021 Timothy Wolff-Piggott
+# This software is distributed under the terms of the MIT license
+# which is available at https://opensource.org/licenses/MIT
+
+import logging
+import os
+
+import mlflow
+
+from hpoflow.optuna_mlflow import OptunaMLflow
+from optuna._imports import try_import
+
+with try_import() as _imports:
+    import transformers
+_imports.check()
+
+_logger = logging.getLogger(__name__)
+
+
+class OMLflowCallback(transformers.TrainerCallback):
+    """
+    Class based on `transformers.TrainerCallback`; integrates with OptunaMLflow
+    to send the logs to `MLflow` and `Optuna` during model training.
```

Comment:
I think this should be RST and they use 2 backticks - not just one.

So something like this:
```
    Class based on ``transformers.TrainerCallback``; integrates with OptunaMLflow
    to send the logs to ``MLflow`` and ``Optuna`` during model training.
```

Also see here for example: https://github.com/optuna/optuna/blob/99484054fcec97eb530960a8612c3e6058ee5690/optuna/pruners/_median.py#L58